README.md

```diff
@@ -0,0 +1,64 @@
+# graphql-auto-requester-datasource
+
+This module provides the graphql-auto-requester as an [Apollo
+DataSource](https://www.apollographql.com/docs/apollo-server/data/data-sources/).
+
+## Creating a DataSource
+There are a few ways to create the DataSource from this library:
+
+###  Using Introspection
+
+If you wish to use introspection, you can just set the `baseURL` for your service in the constructor:
+```js
+class IntrospectedGraphQLService extends GraphQLAutoRequesterDataSource {
+  constructor() {
+    this.baseURL = 'https://example.com/'
+  }
+}
+```
+
+###  Using A Schema Document
+
+If you wish to use a Schema Document, you must set the `baseURL` for your service in the constructor, and provide the
+schema document as a property on the class itself.
```

Comment:
```suggestion
schema document as a property on the class itself:
```

Reply:
You previously used a colon at the end of your other "If you wish to..." statement.